modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -230,14 +230,17 @@ For more information about creating an account with only the required privileges
 [id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
 == Using {product-title} with vMotion
 
-[IMPORTANT]
-====
-{product-title} generally supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.
-====
+If you intend on using vMotion in your vSphere environment, consider the following before installing a {product-title} cluster.
 
-If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. These references prevent affected pods from starting up and can result in data loss.
+* {product-title} generally supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.
```

Comment:
Although I did see [this comment](https://github.com/openshift/openshift-docs/pull/51056#discussion_r984645845), yet the term "generally" still sounds ambiguous in the context of support status. Leave it to your discretion to omit it or not.
```suggestion
* {product-title} supports compute-only vMotion. Using Storage vMotion can cause issues and is not supported.
```

Reply:
@abhatt-rh I am going to hang on to this language.